src/playerCommands.ts

```diff
@@ -841,6 +843,39 @@ ${highestVotedMaps.map(({key:map, value:votes}) =>
 			}
 		};
 	}),
+	surrender : command(() => {
+	
+	function init_manager(){
+		//256 vote managers, cuz why not>
+		let managers = []
+		for (let i = 0; i < 256; i++) {
+			managers.push(new VoteManager<number>(1.5 * 60_000, Gamemode.hexed() ? 1 : undefined, Team.get(i))
+				.on("success", () => {
+					Team.get(i).data().cores.each((core) => {core.kill()});
+				})
+				.on("vote passed", (t) => Call.sendMessage(`[orange]Surrender[white]: Team ${t.team!.coloredName()} has voted to forfeited this match.`))
+				.on("vote failed", (t,_votes,_required,canidates) => {canidates.each((player) => {player.sendMessage(`[orange]Surrender[white]: Team ${t.team!.coloredName()} has chosen not to forfit this match.`)})})
+				.on("player vote change", (t, _fishplayer, oldVote, newVote, canidates) => {canidates.each((player) => {player.sendMessage(`[orange]Surrender[white]: ${player.name}[white] ${oldVote == newVote ? "still " : ""}wants to forfeit this match. [orange]${t.currentVotes()}[white] votes, [orange]${t.requiredVotes()}[white] required.`)})})
+				.on("player vote removed", (t, fishplayer, _previous, canidates) => {canidates.each((player) => {player.sendMessage(`[orange]Surrender[white]: Player ${fishplayer.name}[white] has left the game. [orange]${t.currentVotes()}[white] votes, [orange]${t.requiredVotes()}[white] required.`)})})
+				)
+		
+		}
+		return managers
+	}
+		
+	return{
+		args: [],
+		description: 'Vote to surrender to enemy team',
+		perm: Perm.play,
+		requirements: [Req.cooldown(30_000), Req.modeNot("attack"), Req.modeNot("survival"),],
+		init: () => ({
```

Comment:
This returns an init thunk from the init thunk, idk if I handled that in the commands framework
You can just write data: { manager: init_manager() } as this is already inside an init thunk